prismic-model/js/manual-promos.js

```diff
@@ -0,0 +1,15 @@
+import structuredText from './parts/structured-text';
+import image from './parts/image';
+import link from './parts/link';
+
+const ManualPromo = {
+  Contact: {
+    title: structuredText('Title', 'single', ['heading1']),
+    format: link('Format', 'document', ['article-formats', 'event-formats']),
```

Comment:
I can see the taxonomy of events and articles being hijacked here. Have we checked with the editorial team as to whether we might create a more generic `formats` or `labels` type that might be used here so the clean taxonomy of those types aren't polluted to achieve what they want?

Reply:
Saying that, I imagine this is actually `labels` unless we are only ever going to use it for formats.

Reply:
Just checking I understand you correctly, are you thinking something more like this:

`const Card = {`
` title: structuredText('Title', 'single', ['heading1']),`
` labels: link('Labels', 'document', ['labels']),`
` description: structuredText('Description', 'single'),`
` image: image('Image'),`
` link: link('Link'),`
`};`

and having a new 'labels' content type in Prismic?

Reply:
I think title has it's own type, either way.

> and having a new 'labels' content type in Prismic?

That was only a thought - probably worth talking to the content creators. If we give them a system that doesn't suffice, they'll hijack the other one, that is serving it's specific purpose quite well.

If it is formats, then great, but I imagine there might be things outside of that list, which we can cater for by: `format: link('Format', 'document', ['article-formats', 'event-formats', 'labels'])`.

But if they just want what's there, then great.